config/environments/development.rb

```diff
@@ -34,7 +34,7 @@
   end
 
   # Store uploaded files on the local file system (see config/storage.yml for options).
-  config.active_storage.service = :local
+  config.active_storage.service = :amazon
```

Comment:
We usually configure `:local` for `non-production` environments and `:amazon` for `production` environment

Reply:
Here's an example for a production environment:
https://github.com/rootstrap/rails_api_base/blob/master/config/environments/production.rb#L46

Here's an example for a development environment:
https://github.com/rootstrap/rails_api_base/blob/master/config/environments/development.rb#L37

Here's an example for a test environment configuration:
https://github.com/rootstrap/rails_api_base/blob/master/config/environments/test.rb#L37